boards/amd/versalnet_rpu/doc/index.rst

```diff
@@ -0,0 +1,79 @@
+.. zephyr:board:: versalnet_rpu
+
+Overview
+********
+This configuration provides support for the RPU(R52), real-time processing unit on Xilinx
+Versal Net SOC, it can operate as following:
+
+* Two independent R52 cores with their own TCMs (tightly coupled memories)
+* Or as a single dual lock step unit with the TCM.
+
+This processing unit is based on an ARM Cortex-R52 CPU, it also enables the following devices:
+
+* ARM GIC v3 Interrupt Controller
+* Global Timer Counter
+* SBSA UART
+
+Hardware
+********
+Supported Features
+==================
+
+The following hardware features are supported:
+
++--------------+------------+----------------------+
+| Interface    | Controller | Driver/Component     |
++==============+============+======================+
+| GIC          | on-chip    | generic interrupt    |
+|              |            | controller           |
++--------------+------------+----------------------+
+| Global Timer | on-chip    | system timer         |
++--------------+------------+----------------------+
+| SBSA UART    | on-chip    | serial port          |
++--------------+------------+----------------------+
+
+The kernel currently does not support other hardware features on this platform.
+
+Devices
+========
+System Timer
+------------
+
+This board configuration uses a system timer tick frequency of 100 MHz.
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+on-chip UART0.
+
+Memories
+--------
+
+Although Flash, DDR and OCM memory regions are defined in the DTS file,
+all the code plus data of the application will be loaded in the sram0 region,
+which points to the DDR memory. The ocm0 memory area is currently available
+for usage, although nothing is placed there by default.
+
+Known Problems or Limitations
+==============================
+
+The following platform features are unsupported:
+
+* Only the first core of the R52 subsystem is supported.
+
+Programming and Debugging
+*************************
+
+To download the Zephyr Executable and Linkable Format .elf file, please use following west command.
+
+.. code-block:: console
+
+        west flash --runner xsdb --elf-file versalnet_rpu/zephyr/zephyr.elf
```

Comment:
would be better to use the `zephyr-app-commands` way

Reply:
could you please provide a reference example

Reply:
https://github.com/zephyrproject-rtos/zephyr/blame/main/boards/adi/eval_adin2111ebz/doc/index.rst#L142